content/applications/services/field_service/planning_itinerary.rst

```diff
@@ -0,0 +1,51 @@
+=====================
+Planning an Itinerary
+=====================
+
+By default, **Odoo Field Service** shows a static map where all task locations for the day are
+pinned. To make it more useful for the field service workers, it is possible to display an itinerary
+on the map. To do so enable the **Map Route** feature as follows.
+
+Configure Route Map
+===================
+
+To enable the **Map Routes feature**, a free **MapBox** account is required and needs to be linked
+to Odoo.
+
+Go to the `Mapbox website <https://www.mapbox.com/>`_ :
+ - create an account or log in if you already have one
+ - create a token
+
+Go to the :guilabel:`Settings` app, scroll down to :menuselection:`Integrations --> Map Routes -->
+Token --> paste your MapBox token --> Save`.
```

Comment:
```suggestion
#. Create or log in to a Mapbox account using the following link: `<https://www.mapbox.com/>`_.
#. Create a token: Click :guilabel:`Create a token`, then, in the form that opens, add a :guilabel:`Token name`, scroll down to enter your Odoo databse's :guilabel:`URL`, and enter your password.
#. Copy the access token.
#. In Odoo, open the :guilabel:`Settings` app, scroll down to :guilabel:`Integrations` section, paste your Mapbox access token in the :guilabel:`Token` field under :guilabel:`Map Routes`, and click :guilabel:`Save`.
```
- When you have several steps to perform 1 action (i.e. enable the feature here), it's better to structure it in a numbered list. 
- I would personnally explain how to create an access token since we do it for many other features such as payment providers, but we could also send people to this doc: https://docs.mapbox.com/help/getting-started/access-tokens/#adding-url-restrictions-to-access-tokens if you prefer. In any case, we need to give more info about this.
- menuselection = for menu items exclusively
- We really need to check if this feature is only used for field service, otherwise this configuration part should go in another part of the documentation